src/server/pfs/fuse/loopback.go

```diff
@@ -566,21 +560,29 @@ func (n *loopbackNode) download(path string, state fileState) (retErr error) {
 	// Note, len(parts) < 1 should not actually be possible, but just in case
 	// no need to panic.
 	if len(parts) < 1 || parts[0] == "" {
-		return nil //already downloaded in downloadRepos
+		return nil // already downloaded in downloadRepos
 	}
-	branch := n.root().branch(parts[0])
-	commit, err := n.commit(parts[0])
+	name := parts[0]
+	branch := n.root().branch(name)
+	commit, err := n.commit(name)
 	if err != nil {
 		return err
 	}
 	if commit == "" {
 		return nil
 	}
-	if err := n.c().ListFile(client.NewCommit(parts[0], branch, commit), pathpkg.Join(parts[1:]...), func(fi *pfs.FileInfo) (retErr error) {
+	ro, ok := n.root().repoOpts[name]
+	if !ok {
+		// not ready or something?
+		// XXX should this be an error?
```

Comment:
Might make sense to either return or log an error

Reply:
fixed